.circleci/config.yml

```diff
@@ -376,6 +397,14 @@ workflows:
             - test_unit
             - test_karma
 
+      - test_integration_dev:
+          context: lwc-sauce-labs
```

Comment:
Why is the `lwc-sauce-labs` context is needed here when `test_integration_dev` doesn't use it?

Reply:
Fixed!